### ISSUE TITLE
Add pid to active span table?

### ISSUE THREAD

**Issue by tsloughter** [open-telemetry/opentelemetry-erlang#19]
As I was thinking about adding a `monitor` option to spans so that they can be ended if the process dies I realized it was probably a good idea to also add it to the active table so the sweeper can make a judgement on it for spans not monitored.

The sweeper could then be configured to either ignore expired spans if the process that created it is still alive or to only care about if the process that created it is dead. Meaning, expiration of infinity but if the pid is no longer alive it will sweep it.

Thoughts?

**Comment by hauleth** [comment]
It seems that each span would start two monitors. Is that right? That would cause hell lot of monitors and messages in some applications.

**Comment by tsloughter** [comment]
No, it would be optional. Person would include an option `monitor` when calling `start_span`.

**Comment by tsloughter** [comment]
Oh, and definitely not "two". The pid stored in the active spans table is not a monitor. It is just used by the sweeper.
